src/components/guide-steps.jsx

```diff
@@ -0,0 +1,237 @@
+import React from 'react';
+
+const steps = [
+  {
+    content: (
+      <div>
+        <h2>Welcome to catwalk!</h2>
+        <p>
+          catwalk lets you explore your data model to gain
+          insights about fields, associations and how interactions
+          with the data impacts the model.
+        </p>
+        <p>
+          Follow the guide to discover the power of catwalk.
+        </p>
+      </div>
+    ),
+    placement: 'center',
+    target: 'body',
+  },
+  {
+    content: (
+      <div>
+        <h2>Welcome to catwalk!</h2>
+        <p>The URL to catwalk is</p>
+        <span className="breakword">
+          {window.location.href}
+        </span>
+        <p>
+          where
+          {' '}
+          <code>engine_url</code>
+          {' '}
+          points to the app containing the data model. Change this to explore another data model.
+        </p>
+      </div>
+    ),
+    placement: 'center',
+    target: 'body',
+  },
+  {
+    content: (
+      <div>
+        <p>
+          The highlighted area represents a table in the data model.
+        </p>
+        <p>On the top we can see the table name, together with the number of rows in the table.</p>
+      </div>
+    ),
+    placement: 'right-start',
+    target: '.column',
+    title: 'Table',
+  },
+  {
+    content: (
+      <div>
+        <p>
+          Fields, the data-carrying entities in the data model, are represented with a box like this.
+        </p>
+        <p>
+          The field name and the number of field values are visible.
+        </p>
+        <p>
+          The number of field values are presented in the form of 6 of 6(5)
```

Comment:
This section reads a bit weird for me, shouldn't the full form be presented first before talking about parts of it? I.e. explain `6(5)` before explaining `6`. I also got a bit confused when the numbers weren't related to my app/the focused field :)